src/System.Data.Common/tests/System/Data/DataTableTest2.cs

```diff
@@ -952,16 +900,16 @@ public void OnColumnChanging()
             Assert.False(_eventRaised);
             _eventRaised = false;
             _eventValues = false;
-            dt.ColumnChanging += new DataColumnChangeEventHandler(OnColumnChanging_Handler);
+            dt.ColumnChanging += new DataColumnChangeEventHandler(DataColumnChangeHandler);
             dt.OnColumnChanging_Test();
             // OnColumnChanging Event 2
             Assert.True(_eventRaised);
             // OnColumnChanging Values
             Assert.True(_eventValues);
-            dt.ColumnChanging -= new DataColumnChangeEventHandler(OnColumnChanging_Handler);
+            dt.ColumnChanging -= new DataColumnChangeEventHandler(DataColumnChangeHandler);
         }
 
-        private void OnColumnChanging_Handler(object sender, DataColumnChangeEventArgs e)
+        private void DataColumnChangeHandler(object sender, DataColumnChangeEventArgs e)
```

Comment:
Let's leave "Changing" in the naming (i.e. `DataColumnChangingHandler` not `DataColumnChangeHandler`), since that is the event to which it's added/removed. Same for the other event handlers (e.g. RowChangeHandler)

Reply:
it is added both for `...Changed` and `...Changing` events, and I did not want to suggest they are only for `...Changing` events; They had the exact same content, so it seemed pointless to have separate handlers for each. Same for the other handlers.